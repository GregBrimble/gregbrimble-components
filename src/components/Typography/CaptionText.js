import React from 'react';

import styled from 'styled-components';
import ThemeDefault from "../../themes/ThemeDefault";

const CaptionText = styled.span`
    color: ${(props) => props.theme.neutral['900']};
    font-family: 'Open Sans', sans-serif;
    font-size: 0.75rem;
    font-weight: 400;
    line-height: 1.25rem;
`;

CaptionText.defaultProps = {
    theme: ThemeDefault
};

export default CaptionText;
