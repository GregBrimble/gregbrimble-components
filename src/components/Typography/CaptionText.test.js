import React from "react"
import renderer from "react-test-renderer"
import CaptionText from "./CaptionText"

describe("CaptionText", () => {
  it("renders properly", () => {
    const tree = renderer
      .create(<CaptionText>Caption Text</CaptionText>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
