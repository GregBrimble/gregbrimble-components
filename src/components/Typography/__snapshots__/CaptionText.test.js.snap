// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaptionText renders properly 1`] = `
<span
  className="sc-bdVaJa jeDJSr"
>
  Caption Text
</span>
`;
